store/blocks/pdp/product.jsonc

```diff
@@ -2,7 +2,7 @@
   "store.product": {
     "children": [
       "flex-layout.row#product-breadcrumb",
-      "flex-layout.row#product-main",
+      "condition-layout.product#unavailable",
```

Comment:
Nit: I'd change the id to `#availability`

Reply:
Hi @kaisermann , done